examples/ts/proxy/server.ts

```diff
@@ -1,29 +1,8 @@
-import { Request, Response, NextFunction } from 'express';
-import express = require('express');
-const app = express();
-const { createProxyMiddleware } = require('http-proxy-middleware');
+import { createProxy } from 'proxy';
+import { AddressInfo } from 'net';
+const proxy = createProxy();
 
-// This *must* match the environment you are using in the create-wallet script
-const bitgoApi = 'https://app.bitgo-test.com';
-// TODO: replace with your access token
-const secretAccessToken = 'yourAccessToken';
```

Comment:
Why are we changing this example? The goal of this example is to show an example of the server injecting the access token opposed to needing it in the frontend.

Reply:
I have roll-back the changes to this example and added another example for http proxy usage.
Got confused because of the name. The use case are actually different.